MDANSE_GUI/Src/MDANSE_GUI/InputWidgets/AtomTransmutationWidget.py

```diff
@@ -160,10 +175,26 @@ class AtomTransmutationWidget(AtomSelectionWidget):
 
     _push_button_text = "Atom transmutation helper"
     _default_value = "{}"
-    _tooltip_text = "Specify the atom transmutation that will be used in the analysis. The input is a JSON string, and can be created using the helper dialog."
+    _tooltip_text = (
+        "Specify the atom transmutation that will be used in the analysis."
+        " The input is a JSON string, and can be created using"
+        " the helper dialog."
+    )
+
+    def __init__(self, *args, _use_list_view=True, **kwargs):
+        """Create the main widget for transmuting atom types.
+
+        Parameters
+        ----------
+        _use_list_view : bool, optional
+            If True, a ListView will replace LineEdit, by default True
 
-    def create_helper(self, traj_data: tuple[str, Trajectory]) -> TransmutationHelper:
         """
+        super().__init__(*args, use_list_view=False, **kwargs)
```

Comment:
Slightly concerned that this ignores the `_use_list_view` explicitly. Why is it in the args in that case?